check/base.lua

```diff
@@ -427,14 +427,14 @@ function ChildCheck:_runChild(exePath, exeArgs, environ, callback)
 
   self._log(logging.DEBUG, fmt("%s: starting process", exePath))
 
-  local child = childprocess.spawn(exePath, exeArgs, { env = environ })
+  local child = childprocess.spawn(exePath, exeArgs, { env = environ, detached = true})
   child.stdin:destroy() -- close stdin for windows and ruby compatibility
 
   local pluginTimeout = timer.setTimeout(self._timeout, function()
     local timeoutSeconds = (self._timeout / 1000)
 
     self._log(logging.DEBUG, fmt("%s: Plugin didn't finish in %s seconds, killing it...", exePath, timeoutSeconds))
-    child:kill('sigkill')
+    uv.kill("-" .. child.pid, 9)
```

Comment:
This seems like it might be what was missing. Where did you find that it needs to be prefixed with a hyphen?

Reply:
I have added the source article link in the PR description

Reply:
Oh I see. It's in the general process management one. I thought it was something specific to `uv`.